usagov/usagov-2021
# Usagov 1690 add class to site banner for analytics

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1690

## Description

I created a new twig template for the site banners to add the attribute `data-analytics="site-banner"` to the element.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


1. Create a site banner (You can find the steps on how to do it here: docs/SiteBanner.md or https://docs.google.com/document/d/1kB3H3fnC_QEt9zwil_XZVLM-xWW94erLF0tILfre7hA/edit#heading=h.z6ne0og04bp5)
2. Inspect the site banner that you just created and check that it has this attribute `data-analytics="site-banner"`

### Requires New Config

### Requires New Content

### Validation Steps

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
4. Find the commit of this pull request.
5. Build and deploy the changes.
6. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
